Build failures

Trying to build via the README and it keeps failing.

Dockerfile
```
FROM ubuntu:20.04

RUN apt update
RUN DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata
RUN apt install -y git build-essential cmake docutils-common libelfin-dev nodejs npm python3 python3-pip
RUN pip install conan
RUN git clone https://github.com/plasma-umass/coz.git \
    && mkdir coz/build \
    && cd coz/build \
    && conan install .. \
    && cmake ..
```

```
> docker build -t foo .
[+] Building 596.8s (9/9) FINISHED
 => [internal] load build definition from Dockerfile                                                                     0.0s
 => => transferring dockerfile: 573B                                                                                     0.0s
 => [internal] load .dockerignore                                                                                        0.0s
 => => transferring context: 2B                                                                                          0.0s
 => [internal] load metadata for docker.io/library/ubuntu:20.04                                                          1.0s
 => [1/6] FROM docker.io/library/ubuntu:20.04@sha256:cf31af331f38d1d7158470e095b132acd126a7180a54f263d386da88eb681d93    0.0s
 => CACHED [2/6] RUN apt update                                                                                          0.0s
 => CACHED [3/6] RUN DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata                                          0.0s
 => [4/6] RUN apt install -y git build-essential cmake docutils-common libelfin-dev nodejs npm python3 python3-pip     562.6s
 => [5/6] RUN pip install conan                                                                                         11.5s
 => ERROR [6/6] RUN git clone https://github.com/plasma-umass/coz.git     && mkdir coz/build     && cd coz/build     &  21.7s
------
 > [6/6] RUN git clone https://github.com/plasma-umass/coz.git     && mkdir coz/build     && cd coz/build     && conan install ..     && cmake ..:
#9 0.489 Cloning into 'coz'...
#9 2.600 Auto detecting your dev setup to initialize the default profile (/root/.conan/profiles/default)
#9 2.603 Found gcc 9
#9 2.606 gcc>=5, using the major as version
#9 2.606
#9 2.606 ************************* WARNING: GCC OLD ABI COMPATIBILITY ***********************
#9 2.606
#9 2.606 Conan detected a GCC version > 5 but has adjusted the 'compiler.libcxx' setting to
#9 2.606 'libstdc++' for backwards compatibility.
#9 2.606 Your compiler is likely using the new CXX11 ABI by default (libstdc++11).
#9 2.606
#9 2.606 If you want Conan to use the new ABI for the default profile, run:
#9 2.606
#9 2.606     $ conan profile update settings.compiler.libcxx=libstdc++11 default
#9 2.606
#9 2.606 Or edit '/root/.conan/profiles/default' and set compiler.libcxx=libstdc++11
#9 2.606
#9 2.606 ************************************************************************************
#9 2.606
#9 2.606
#9 2.606
#9 2.606 Default settings
#9 2.606        os=Linux
#9 2.606        os_build=Linux
#9 2.606        arch=x86_64
#9 2.606        arch_build=x86_64
#9 2.606        compiler=gcc
#9 2.606        compiler.version=9
#9 2.606        compiler.libcxx=libstdc++
#9 2.606        build_type=Release
#9 2.607 *** You can change them in /root/.conan/profiles/default ***
#9 2.607 *** Or override with -s compiler='other' -s ...s***
#9 2.607
#9 2.607
#9 2.654 Configuration:
#9 2.654 [settings]
#9 2.654 arch=x86_64
#9 2.654 arch_build=x86_64
#9 2.654 build_type=Release
#9 2.654 compiler=gcc
#9 2.654 compiler.libcxx=libstdc++
#9 2.654 compiler.version=9
#9 2.654 os=Linux
#9 2.654 os_build=Linux
#9 2.654 [options]
#9 2.654 [build_requires]
#9 2.654 [env]
#9 2.654
#9 3.254 libelfin/0.3: Not found in local cache, looking in remotes...
#9 3.254 libelfin/0.3: Trying with 'conan-center'...
#9 3.268 Downloading conanmanifest.txt
#9 3.280 Downloading conanfile.py
#9 3.293 Downloading conan_export.tgz
#9 3.314 libelfin/0.3: Downloaded recipe revision 0
#9 19.99 bzip2/1.0.8: Not found in local cache, looking in remotes...
#9 19.99 bzip2/1.0.8: Trying with 'conan-center'...
#9 20.05 Downloading conanmanifest.txt
#9 20.06 Downloading conanfile.py
#9 20.07 Downloading conan_export.tgz
#9 20.09 bzip2/1.0.8: Downloaded recipe revision 0
#9 20.09 sqlite3/3.35.5: Not found in local cache, looking in remotes...
#9 20.09 sqlite3/3.35.5: Trying with 'conan-center'...
#9 20.11 Downloading conanmanifest.txt
#9 20.12 Downloading conanfile.py
#9 20.13 Downloading conan_export.tgz
#9 20.14 sqlite3/3.35.5: Downloaded recipe revision 0
#9 20.20 Version ranges solved
#9 20.20     libelfin/* versions found in 'conan-center' remote
#9 20.20     Version range '>=0.3' required by 'conanfile.txt' resolved to 'libelfin/0.3' in remote 'conan-center'
#9 20.20     bzip2/* versions found in 'conan-center' remote
#9 20.20     Version range '>=1.0.8' required by 'conanfile.txt' resolved to 'bzip2/1.0.8' in remote 'conan-center'
#9 20.20     sqlite3/* versions found in 'conan-center' remote
#9 20.20     Version range '>=3.33.0' required by 'conanfile.txt' resolved to 'sqlite3/3.35.5' in remote 'conan-center'
#9 20.20
#9 20.20 conanfile.txt: Installing package
#9 20.20 Requirements
#9 20.20     libelfin/0.3 from 'conan-center' - Downloaded
#9 20.20 Packages
#9 20.20     libelfin/0.3:50a5030bbbb13ae56bc4be41915ecd48549cb895 - Download
#9 20.20 Build requirements
#9 20.20     bzip2/1.0.8 from 'conan-center' - Downloaded
#9 20.20     sqlite3/3.35.5 from 'conan-center' - Downloaded
#9 20.20 Build requirements packages
#9 20.20     bzip2/1.0.8:da606cf731e334010b0bf6e85a2a6f891b9f36b0 - Download
#9 20.20     sqlite3/3.35.5:c0cc1a2703aaeb2777d683089def26db57cf15e6 - Download
#9 20.20
#9 20.20 Installing (downloading, building) binaries...
#9 20.20 bzip2/1.0.8: Retrieving package da606cf731e334010b0bf6e85a2a6f891b9f36b0 from remote 'conan-center'
#9 20.24 Downloading conanmanifest.txt
#9 20.25 Downloading conaninfo.txt
#9 20.27 Downloading conan_package.tgz
#9 20.30 bzip2/1.0.8: Package installed da606cf731e334010b0bf6e85a2a6f891b9f36b0
#9 20.30 bzip2/1.0.8: Downloaded package revision 0
#9 20.30 libelfin/0.3: Retrieving package 50a5030bbbb13ae56bc4be41915ecd48549cb895 from remote 'conan-center'
#9 20.33 Downloading conanmanifest.txt
#9 20.34 Downloading conaninfo.txt
#9 20.35 Downloading conan_package.tgz
#9 20.39 libelfin/0.3: Package installed 50a5030bbbb13ae56bc4be41915ecd48549cb895
#9 20.39 libelfin/0.3: Downloaded package revision 0
#9 20.39 sqlite3/3.35.5: Retrieving package c0cc1a2703aaeb2777d683089def26db57cf15e6 from remote 'conan-center'
#9 20.42 Downloading conanmanifest.txt
#9 20.43 Downloading conaninfo.txt
#9 20.45 Downloading conan_package.tgz
#9 20.71 sqlite3/3.35.5: Package installed c0cc1a2703aaeb2777d683089def26db57cf15e6
#9 20.71 sqlite3/3.35.5: Downloaded package revision 0
#9 20.71 bzip2/1.0.8: Appending PATH environment variable: /root/.conan/data/bzip2/1.0.8/_/_/package/da606cf731e334010b0bf6e85a2a6f891b9f36b0/bin
#9 20.71 sqlite3/3.35.5: Appending PATH env var with : /root/.conan/data/sqlite3/3.35.5/_/_/package/c0cc1a2703aaeb2777d683089def26db57cf15e6/bin
#9 20.71 conanfile.txt: Applying build-requirement: bzip2/1.0.8
#9 20.71 conanfile.txt: Applying build-requirement: sqlite3/3.35.5
#9 20.71 conanfile.txt: Generator txt created conanbuildinfo.txt
#9 20.71 conanfile.txt: Generator cmake_find_package created FindBZip2.cmake
#9 20.71 conanfile.txt: Generator cmake_find_package created FindSQLite3.cmake
#9 20.71 conanfile.txt: Generator cmake_find_package created Findlibelfin.cmake
#9 20.72 conanfile.txt: Generated conaninfo.txt
#9 20.72 conanfile.txt: Generated graphinfo
#9 20.72 WARN: Remotes registry file missing, creating default one in /root/.conan/remotes.json
#9 20.82 -- The C compiler identification is GNU 9.3.0
#9 20.89 -- The CXX compiler identification is GNU 9.3.0
#9 20.90 -- Check for working C compiler: /usr/bin/cc
#9 20.97 -- Check for working C compiler: /usr/bin/cc -- works
#9 20.97 -- Detecting C compiler ABI info
#9 21.05 -- Detecting C compiler ABI info - done
#9 21.07 -- Detecting C compile features
#9 21.07 -- Detecting C compile features - done
#9 21.07 -- Check for working CXX compiler: /usr/bin/c++
#9 21.16 -- Check for working CXX compiler: /usr/bin/c++ -- works
#9 21.16 -- Detecting CXX compiler ABI info
#9 21.25 -- Detecting CXX compiler ABI info - done
#9 21.26 -- Detecting CXX compile features
#9 21.27 -- Detecting CXX compile features - done
#9 21.27 -- Looking for pthread.h
#9 21.35 -- Looking for pthread.h - found
#9 21.35 -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
#9 21.43 -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
#9 21.43 -- Looking for pthread_create in pthreads
#9 21.49 -- Looking for pthread_create in pthreads - not found
#9 21.49 -- Looking for pthread_create in pthread
#9 21.58 -- Looking for pthread_create in pthread - found
#9 21.58 -- Found Threads: TRUE
#9 21.58 -- Found libelfin: 0.3 (found version "0.3")
#9 21.58 CMake Error at libcoz/CMakeLists.txt:35 (install):
#9 21.58   install TARGETS given no LIBRARY DESTINATION for module target "coz".
#9 21.58
#9 21.58
#9 21.58 CMake Error at libcoz/CMakeLists.txt:39 (install):
#9 21.58   install DIRECTORY given no DESTINATION!
#9 21.58
#9 21.58
#9 21.59 -- Configuring incomplete, errors occurred!
#9 21.59 See also "/coz/build/CMakeFiles/CMakeOutput.log".
#9 21.59 See also "/coz/build/CMakeFiles/CMakeError.log".
------
executor failed running [/bin/sh -c git clone https://github.com/plasma-umass/coz.git     && mkdir coz/build     && cd coz/build     && conan install ..     && cmake ..]: exit code: 1
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,6 +16,13 @@ add_compile_options(-gdwarf-3)
 
 option(INSTALL_COZ "Enable installation of coz. (Projects embedding coz may want to turn this OFF.)" ON)
 
+if(INSTALL_COZ)
+    include(GNUInstallDirs)
+    install(PROGRAMS coz DESTINATION bin)
+    install(FILES LICENSE.md DESTINATION licenses)
+    install(FILES cmake/coz-profilerConfig.cmake DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+endif()
+
 add_subdirectory(libcoz)
 
 option(BUILD_BENCHMARKS "Build benchmarks" OFF)
@@ -28,9 +35,3 @@ if(BUILD_BENCHMARKS)
     add_subdirectory(benchmarks)
 endif()
 
-if(INSTALL_COZ)
-    include(GNUInstallDirs)
-    install(PROGRAMS coz DESTINATION bin)
-    install(FILES LICENSE.md DESTINATION licenses)
-    install(FILES cmake/coz-profilerConfig.cmake DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
-endif()

